src/components/GWL.vue

```diff
@@ -211,9 +211,9 @@ export default {
         // read in data 
         let promises = [
         self.d3.csv(self.publicPath + "quant_peaks.csv",  this.d3.autotype), // used to draw legend shapes - color palette needs to be pulled out
-        self.d3.csv("https://labs.waterdata.usgs.gov/visualizations/data/gw-conditions-peaks-timeseries.csv",  this.d3.autotype),
+        self.d3.csv("https://labs.waterdata.usgs.gov/visualizations/data/gw-conditions-peaks-timeseries-wy22.csv",  this.d3.autotype),
         self.d3.csv("https://labs.waterdata.usgs.gov/visualizations/data/gw-conditions-site-coords.csv",  this.d3.autotype), 
-        self.d3.csv("https://labs.waterdata.usgs.gov/visualizations/data/gw-conditions-daily-proportions.csv",  this.d3.autotype),
+        self.d3.csv("https://labs.waterdata.usgs.gov/visualizations/data/gw-conditions-daily-proportions-wy22.csv",  this.d3.autotype),
```

Comment:
This seems like a good change to make, but should there be a corresponding change to [the filename](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/3_visualize.yml#L42) in `3_visualize.yml`? Are you making the edit manually right now? I see you make [a copy of the file](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/3_visualize.yml#L77-L80) with the dates in the filename further down. Should that be used here instead? And should [this comment](https://github.com/USGS-VIZLAB/gw-conditions/blob/main/3_visualize.yml#L73-L75) be updated to note that the file will need to be manually updated in `GWL.vue`?

Reply:
Yes you are absolutely correct. However, this puts the data in the dev s3 bucket which cannot be accessed through a url in the browser. I did this manually as a band-aid to get the correct data showing for the last WY. We need to refine this piece, also to consider how we will set up to be able to change the dates to regenerate each month.

Reply:
We could redirect this to the public bucket instead of the dev as a solution. Definitely don't need to keep both. Also should consider how long we should store these updates once we have a routine down

Reply:
Cool, sounds good. I'll make an issue with these notes, and then this is good to merge

Reply:
see #80